hunts/src/store.js

```diff
@@ -12,5 +18,13 @@ export default configureStore({
     puzzles: puzzlesReducer,
     hunt: huntReducer,
     tagFilter: tagFilterReducer,
+    filter: filterReducer,
   },
+  middleware: (getDefaultMiddlewares) => [
+    ...getDefaultMiddlewares(),
+    save({
+      states: ["filter.solveStateFilter"],
```

Comment:
I see you are saving just the solveStateFilter here. What do you think about saving the getSolveStateFilter and tagFilter too?

Reply:
I added the tag filter to the persisted data. 

I tried adding the text filter too, but it felt really unnatural to open the page and have that value set, so I omitted it.

Reply:
That's fair! Thanks for adding the tag filter.